behave_testrail_reporter/api.py

```diff
@@ -1,5 +1,6 @@
-import textwrap
+# -*- coding: utf-8 -*-
```

Comment:
As you were discussing in person, I share the strong opinion that `from __future__ import unicode_string` is a more robust solution.